apps/e2e/cypress/e2e/back-office-appeals/reviewAppellantCase.spec.js

```diff
@@ -30,7 +30,8 @@ describe('Appeals feature', () => {
 		appealsListPage.clickReviewAppellantCase(4);
 		appealsListPage.selectRadioButtonByValue('Invalid');
 		appealsListPage.clickButtonByText('Continue');
-		appealsListPage.chooseCheckboxByIndex(1);
+		appealsListPage.chooseCheckboxByIndex(3);
+		appealsListPage.fillInput('Hello here is some extra info, have a nice day 7384_+!£ =');
```

Comment:
If you are testing special characters, I would also suggest some numeric code (&#48;), double quotes ("), and maybe an unsanitized HTML string: (<script>alert(1)<)

Reply:
Thank you, good point, I'll in the next PR i do!